### PR TITLE
Update data-classification-activity-explorer-available-events.md

### PR DESCRIPTION
**Pull request by Bhavanesh**
Made corrections based on customer feedback, other internal reviews

### CODE REVIEW

**Comment by Bhavanesh** [comment]
@chrfox - Can you please review this and approve the changes.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/data-classification-activity-explorer-available-events.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ This event is generated each time a label is removed from a file or document.
 
 ## Sensitivity label file read
 
-This event is generated each time a labeled or protected document is opened.
+This event is generated each time a sensitivity labeled or protected document is opened.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-activity-explorer-available-events.md:95]:
```suggestion
This event is generated each time a sensitivity label or protected document is opened.
```

File: microsoft-365/compliance/data-classification-activity-explorer-available-events.md [hunk 2]
```diff
@@ -17,14 +17,14 @@ ms.collection:
 search.appverid: 
 - MOE150
 - MET150
-description: "listing of labeling actions that are available in activity explorer."
+description: "listing of activities that are available in activity explorer."
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-activity-explorer-available-events.md:20]:
```suggestion
description: "listing of activities that are available in Activity explorer."
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrfox** [comment]
some changes made in a different PR, not changing the title closing

**Pull request closed without merging**